File: HandlowiecApp/app/src/main/java/com/pronova/handlowiec/data/model/DashboardData.kt
package com.pronova.handlowiec.data.model

import com.google.gson.annotations.SerializedName

data class DashboardData(
    @SerializedName("data")
    val data: String = "",

    @SerializedName("sumaZamowien")
    val sumaZamowien: Double = 0.0,

    @SerializedName("liczbaZamowien")
    val liczbaZamowien: Int = 0,

    @SerializedName("liczbaKlientow")
    val liczbaKlientow: Int = 0,

    @SerializedName("sumaPalet")
    val sumaPalet: Int = 0,

    @SerializedName("liczbaAnulowanych")
    val liczbaAnulowanych: Int = 0
)

File: MobileAPI/Models/DashboardData.cs
namespace MobileAPI.Models;

public class DashboardData
{
    public DateTime Data { get; set; }
    public decimal SumaZamowien { get; set; }
    public int LiczbaZamowien { get; set; }
    public int LiczbaKlientow { get; set; }
    public int SumaPalet { get; set; }
    public int LiczbaAnulowanych { get; set; }
}
